recipe/meta.yaml

```
@@ -49,7 +49,7 @@ outputs:
       run_constrained:
         - {{ pin_subpackage("openblas" ~ name_suffix, max_pin="x.x.x") }}
     files:
-      - lib/libopenblas{{ SYMBOLSUFFIX }}p*{{ SHLIB_EXT }}   # [not win]
+      - lib/libopenblas{{ SYMBOLSUFFIX }}*p*{{ SHLIB_EXT }}   # [not win]
```

`*p*` is a bit fragile (as in: quite loose), no?

How about
```suggestion
    # everything but osx-arm64 has DYNAMIC_ARCH and needs no extension
    {% set TARGET_ARCH = "_vortex" if osx-arm64 else "" %}
      - lib/libopenblas{{ SYMBOLSUFFIX }}{{ TARGET_ARCH }}p*{{ SHLIB_EXT }}   # [not win]
```

---

Removed in e12407a

---

nvm, removed that commit

---

I don't think it's loose. It avoids having to have special cases.

---

Fine, just wanted to raise it because it's not very transparent what's happening.

Personally, I'd prefer something more explicit (example below) because these recipes get really hard to read for someone not as well-versed as you are. But also fine if you keep it as is.
```suggestion
      - lib/libopenblas{{ SYMBOLSUFFIX }}p*{{ SHLIB_EXT }}   # [not (win or osx-arm64)]
      - lib/libopenblas{{ SYMBOLSUFFIX }}.so.*               # [linux]
      - lib/libopenblas_vortexp*{{ SHLIB_EXT }}              # [osx-arm64]
```

---

Cool, thanks a lot for the comment!